docs/dev/CodingGuidelines/CodingGuidelines.md

```diff
@@ -21,6 +21,7 @@
     * [Parametrized Diagnostic Messages](#parametrized-diagnostic-messages)
     * [Test Methods](#test-methods)
     * [Async Anonymous Delegates](#async-anonymous-delegates)
+	* [Value Tuples](#value-tuples)
```

Comment:
The original document uses spaces - looks ugly:
![image](https://user-images.githubusercontent.com/1544021/76320275-81a5f980-62f1-11ea-9aca-c97075f8233b.png)

Reply:
Fixed, please check now